inferelator_ng/tfa.py

```diff
@@ -31,12 +31,14 @@ class TFA:
     def __init__(self, prior, exp_mat, exp_mat_halftau):
         self.prior = prior
         self.exp_mat = exp_mat
-        self.exp_mat = exp_mat_halftau
+        self.exp_mat_halftau = exp_mat_halftau
 
     def tfa(self, noself = True, dup_self = True):
         tfwt = self.prior.abs().sum(axis = 0) > 0
         tfwt = tfwt[tfwt].index.tolist()
 
+        dTFs = []
```

Comment:
Ok. Does anyone use the parameter dup_self = False? If not, I think we
should remove the param.

On Wed, Aug 3, 2016 at 2:00 PM, Ren Yi <notifications@github.com> wrote:

> In inferelator_ng/tfa.py
> <https://github.com/ryi06/inferelator_ng/pull/1#discussion_r73383897>:
>
> >
> >      def tfa(self, noself = True, dup_self = True):
> >          tfwt = self.prior.abs().sum(axis = 0) > 0
> >          tfwt = tfwt[tfwt].index.tolist()
> >
> > +        dTFs = []
>
> I agree. I had the same concern before. But Christoph's code didn't take
> into account the situation where dup_self is False.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ryi06/inferelator_ng/pull/1/files/938dcbe260ce87a74b12750838df63e6ca0e9bd5#r73383897>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAGbtme8OeLRJu3QuRbTBEaN5jzAM3mUks5qcNc-gaJpZM4JbvTo>
> .
>